src/pages/display/index.js

```diff
@@ -1062,151 +1065,166 @@ export default class Display extends Component {
 
         {/* TODO - someone really needs to clean up the other tabs :) */}
         {this.state.collabsState && (
-          <CollabsTab wallet={this.state.wallet} filter={this.state.filter} />
+          <CollabsTab
+            wallet={this.state.wallet}
+            filter={this.state.filter}
+            onLoaded={this.disableLoading}
+          />
         )}
-
-        {!this.state.loading && this.state.collectionState && (
-          <div>
+        {((this.state.collabsState && this.state.collabs.length === 0) ||
```

Comment:
collabs length is always 0 because they are loaded in the CollabTab component. this means currently always 'No Collabs' is shown.

<img width="953" alt="Bildschirmfoto 2022-03-23 um 08 09 11" src="https://user-images.githubusercontent.com/182512/159643580-96193e47-406e-4958-a470-3a49501e0576.png">

Reply:
fixed